CMakeLists.txt

```diff
@@ -368,7 +368,9 @@ if(WITH_CORE)
     message (SEND_ERROR "sqlite3 dependency was not found!")
   endif()
 
-  find_package(Protobuf REQUIRED)   # for decoding of vector tiles in MVT format
+  find_package(Protobuf CONFIG REQUIRED)  # for decoding of vector tiles in MVT format
```

Comment:
Maybe a conditional find is required here?

Reply:
CI is green now.